static/js/settings_notifications.js

```diff
@@ -6,6 +8,14 @@ const stream_notification_settings = [
     "wildcard_mentions_notify",
 ];
 
+const stream_specific_notification_settings = [
+    "desktop_notifications",
+    "audible_notifications",
+    "push_notifications",
+    "email_notifications",
+    "wildcard_mentions_notify",
+];
```

Comment:
You can drop this commit, https://github.com/zulip/zulip/pull/14066/commits/d59ddf4970787b794e0b03f30bd2e6cb07487af4 and simply export that variable to use here